docs/Glossary.md

```diff
@@ -0,0 +1,164 @@
+# Definitions
+
+There is seperate section [terminology](./spec/terminology.md) is a separate file additional to this Glossary for the Q&A. The terminology is for advanced to proficient Sidetree users.
+
+## Abbreviations
+In alphabetic order:\
+BX = [Bidirectional model transformation](#bidirectional-model-transformation)
+CAS = Content Addressable Storage 
```

Comment:
for some reasons CAS does not appear on a separate line:
![Screen Shot 2021-05-24 at 11 26 09 AM](https://user-images.githubusercontent.com/7913565/119377527-d9284f00-bc82-11eb-87cf-5402fd96d527.png)

Reply:
added slashes